integration-tests/tasks/check-block-hashes.ts

```diff
@@ -16,13 +17,13 @@ task(
     let netA
     let netB
     try {
-      netA = await providerA.getNetwork()
+      netA = await getChainId(providerA)
```

Comment:
On line 31/32 below we check if `netA.chainId !== netB.chainId` but when `getNetwork()` is replaced by `getChainId()` the response is a number (instead of an object with the `.chainId` property). Will have to modify line 31/32 to reflect this.